pset-functions.js

```diff
@@ -148,6 +182,22 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+function doubleLetters(word) {
+
+  let final = word.split("")
+  let duplicate = word.split("")
+
+  for(i = 1; i < final.length; i++) {
+    if(final[i] === duplicate[i + 1])
```

Comment:
Your logic to this problem provides a great solution! One suggestion: if you get rid of line 188-189 you can substitute the conditional within the if statement on line 192 with (word[i] === word[i + 1]) like in this [example](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L225).